inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java

```diff
@@ -60,8 +61,23 @@ public interface ExtractNodeProvider extends NodeProvider {
      */
     default List<FieldInfo> parseStreamFieldInfos(List<StreamField> streamFields, String nodeId) {
         // Filter constant fields
+        return parseStreamFieldInfos(streamFields, nodeId, null);
```

Comment:
Why is the `dataTypeMappingStrategy` set to empty? I don't think you should do this if you want to use the original fieldtype. There should be a class to handle cases where the current type does not support mapping.

Reply:
> Why is the `dataTypeMappingStrategy` set to empty? I don't think you should do this if you want to use the original fieldtype. There should be a class to handle cases where the current type does not support mapping.

Overload the current method, I think both will work